client/src/pages/Register.js

```diff
@@ -49,6 +49,17 @@ class Register extends Component {
       .post("/api/users", userData)
       .then(response => {
         console.log("success! Data received = ", response.data);
+
+        // get jwt token from header
+        const token = response.headers["x-auth-token"];
+
+        // add token to local storage
+        window.localStorage.setItem("token", token);
```

Comment:
hm! I'm curious - why did you choose to store a token in localstorage? 

at first look my instincts tell me this is probably going to be a security issue down the road. 

a quick reading: https://logrocket.com/blog/jwt-authentication-best-practices/ 
> Don't store it in local storage (or session storage). If any of the 3rd part scripts you include in your page gets compromised, it can access all your users' tokens.

but if this is just to test out retrieving the token and down the road, you'll store it safely!

Reply:
oh wait i noticed this was actually a ticket and it was its original intention to store it on local storage. 

welp! then thats just my two cents. otherwise this seems pretty straight forward and i'll leave it to the team to merge it in :)

Reply:
Yup, you're right but Shums said that localStorage is okay for the scope of this project.